.github/workflows/release.yml

```diff
@@ -5,39 +5,33 @@ on:
     tags: ["v[0-9]+.[0-9]+.[0-9]+*"]
 
 jobs:
-  check:
-    name: Check
+  build:
+    name: Build
     runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        arch: [ "x86_64", "aarch64" ]
     steps:
       - uses: actions/checkout@v3
         with:
           submodules: recursive
       - name: Setup Rust toolchain and cache
         uses: actions-rust-lang/setup-rust-toolchain@v1.3.7
-      - name: Install just
-        uses: taiki-e/install-action@just
+      - uses: taiki-e/install-action@just
       - name: Install requirements
-        run: sudo env PATH=$PATH just ci-prepare
-      - name: Build
-        run: just youki-release
+        if: ${{ matrix.arch == 'x86_64' }}
+        run: | 
+          sudo env PATH=$PATH arch=${{ matrix.arch }} just ci-prepare
+      - name: Install requirements for aarch64
+        if: ${{ matrix.arch == 'aarch64' }}
+        run: cargo install cross --git https://github.com/cross-rs/cross
+      - name: Release build
+        run: arch=${{ matrix.arch }} just youki-release
       - name: test
         # TODO(utam0k): The feature test needs nightly
         # run: just test-basic featuretest test-oci
+        if: ${{ matrix.arch == 'x86_64' }}
```

Comment:
**thought:** With the new arm vms in GitHub Actions we could run the tests in aarch64. Does anyone have any experience with arm vms in GH Actions?

Reply:
I think the arm64 runners are still in private beta, and one has to register and get accepted in order to use them. If you mean running VM in CI and then running the compiling / tests inside the VM ; we had discussed this in an issue a couple of times before, and it seemed to be more pain than use at that time

Reply:
I meant the runners. Would be nice to revisit when/if they are released to the general public.

Reply:
> **thought:** With the new arm vms in GitHub Actions we could run the tests in aarch64. Does anyone have any experience with arm vms in GH Actions?

It seems that the official does not support VMs with built-in aarch64. I have tried https://cirun.io before, but it requires Google Cloud or Microsoft Cloud.